test/import_test.rb

```diff
@@ -202,37 +204,49 @@
       end
     end
   end
-  
+
   context "with an array of columns and an array of unsaved model instances" do
     let(:topics) { Build(2, :topics) }
-    
+
     it "should import records populating the supplied columns with the corresponding model instance attributes" do
       assert_difference "Topic.count", +2 do
         result = Topic.import [:author_name, :title], topics
       end
-      
+
       # imported topics should be findable by their imported attributes
       assert Topic.find_by_author_name(topics.first.author_name)
       assert Topic.find_by_author_name(topics.last.author_name)
     end
-  
+
     it "should not populate fields for columns not imported" do
       topics.first.author_email_address = "zach.dennis@gmail.com"
       assert_difference "Topic.count", +2 do
         result = Topic.import [:author_name, :title], topics
       end
-      
+
       assert !Topic.find_by_author_email_address("zach.dennis@gmail.com")
     end
   end
-  
+
   context "with an array of columns and an array of values" do
     it "should import ids when specified" do
       Topic.import [:id, :author_name, :title], [[99, "Bob Jones", "Topic 99"]]
       assert_equal 99, Topic.last.id
     end
+
+    it "should import data with different encodings" do
```

Comment:
Should this spec fail without the change to `values_sql_for_columns_and_attributes` above? In my tests it always passes.